docs/trace_collection.md

```diff
@@ -92,6 +92,24 @@ class SampleApplication : Application() {
     ```kotlin
     span.setTag("http.url", url)
     ```
+7. Logging a throwable or an error message for a Span:
+
+    Using the Official Opentracing API
+
+    ```kotlin
+    span.log(mutableMapOf(Fields.ERROR_OBJECT to throwable))
+    ```
+    ```kotlin
+    span.log(mutableMapOf(Fields.MESSAGE to errorMessage))
```

Comment:
These don't need to be mutableMap. using `mapOf` will make the sample easier to read

Reply:
It seems you didn't change this doc here ?

Reply:
Sorry my bad....missed that. Done now.